dropwizard-testing/src/main/java/io/dropwizard/testing/POJOConfigurationFactory.java

```diff
@@ -0,0 +1,41 @@
+package io.dropwizard.testing;
+
+import java.io.File;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import io.dropwizard.Configuration;
+import io.dropwizard.configuration.ConfigurationFactory;
+import io.dropwizard.configuration.ConfigurationSourceProvider;
+
+public class POJOConfigurationFactory<C extends Configuration>
+    extends ConfigurationFactory<C>
+{
```

Comment:
Nit: It's more idiomatic to place an opening brace on the same line as the class declaration.